src/dashboard/components/files/FilesHome.js

```diff
@@ -16,7 +16,10 @@ export default function FilesHome(props) {
   const createWithTemplate = (template) => {
     MPQ.push('btn_create_with_template')
     try {
-      createNew(template.templateData)
+      createNew({
+        version: template.version,
```

Comment:
in the case of user templates, the version is a timestamp instead of a version number (e.g. 2021.2.4) so I think we'll need to convert this to a version number first here

Reply:
I actually think that it's pretty good as is.
The timestamp puts the template in a time interval between two versions, so it's valid for the most recent older version of Plottr.
The migrator will treat this as a version and apply only migrations after that date.
This should also work for the older non-timestamp versions of Plottr.

Am I missing something?